src/sentry/spans/consumers/process/factory.py

```diff
@@ -58,45 +60,89 @@ def process_message(message: Message[KafkaPayload]) -> ProduceSegmentContext | N
         return None
 
     assert isinstance(message.value, BrokerValue)
-    try:
-        span = _deserialize_span(message.payload.value)
+    metrics.incr("process_spans.spans.received.count")
+
+    with sentry_sdk.start_transaction(op="process", name="spans.process.process_message") as txn:
+        payload_value = message.payload.value
+        timestamp = int(message.value.timestamp.timestamp())
+        partition = message.value.partition.index
+
+        txn.set_tag("payload", payload_value)
```

Comment:
No payload in tags.

Reply:
good callout, updated https://github.com/getsentry/sentry/pull/68285/commits/03ecd0364b2e9af0c202f2ea7d4edd1e72b9891e